R/Sound_DMR.R

```diff
@@ -1415,11 +1411,12 @@ generate_zoomframe <- function(gene_cord_df, MFrame, Gene_col, target_info=TRUE,
   }
   Final_gene_set <- Final_gene_set[,c(1,(ncol(Final_gene_set)-2),2:4,5:(ncol(Final_gene_set)-3),ncol(Final_gene_set)-1,ncol(Final_gene_set))]
   
-  cat("Zoomframe generated, Adding in target info column; Almost done!\n")
+  message("Zoomframe generated, Adding in target info column; Almost done!")
   
   #Clean up columns with NAs
   Meth_Unmeth <- Final_gene_set %>% select(starts_with("Meth"), starts_with("UnMeth")) %>% colnames()
-  cat("Columns to change NAs -> 0s\n" , Meth_Unmeth)
+  cat("Changing NAs in Methylated and UnMethylated columns to 0\n")
+  cat("Columns :", Meth_Unmeth, "\n" )
```

Comment:
Do we want to use `message()` here as well?

Reply:
Probably want messages to be something that tells what step is running instead of QC steps like this. 
![image](https://github.com/SoundAg/sounDMR/assets/80559286/53f6bce9-f142-468d-a706-effb355ecefb)
Don't have a huge preference to either but formatting is something that should be changed through all the functions
so we discuss about this it then!